modules/debts_api/app/models/debts_api/v0/form5655_submission.rb

```diff
@@ -98,7 +98,9 @@ def register_failure(message)
       end
     end
 
-    def alert_silent_error
+    def alert_silent_error(message)
+      StatsD.increment("#{STATS_KEY}.hard_failure")
+      Rails.logger.error("Silent failure triggered: #{id} - #{message}")
```

Comment:
Can you please verify there's no sensitive information / PII being plugged in this message?

Reply:
We are good to go, there's no PII coming through. 

We've been logging this for a while in another place. I just added this logger here to tightly couple the error logging with the silent error. I want to ensure we are not flagging the Sharepoint errors as silent errors.

Reply:
Thanks